README.md

```diff
@@ -2942,3 +3020,23 @@ IMetadateDelete {
 ```
 
 This method deletes existing metadata for an asset or entry. It accepts metadata configuration as required arguments. This config contains basic details that you need to identify the metadata object you want to delete.
+
+## Modal
+
+The `Modal` class represents a modal dialog opened from the app within the Contentstack UI. It is a powerful feature of the App SDK that enables apps to open modal dialogues, providing an enhanced user experience.
+
+> **Note**: Starting from v1.6.0 of the App SDK, modals now open to take the full screen by default, without any additional user action.
+
+### `setBackgroundElement(element: HTMLElement)`
+
+This method allows developers to specify a custom HTML element to be displayed in the background in place of the app iframe when the modal is opened. By default, the App SDK automatically selects an element to be shown in the background. However, this method provides the flexibility to choose a different element if desired.
```

Comment:
"element to be displayed in the background, in place of .."
"different element if required."

Reply:
Addressed.